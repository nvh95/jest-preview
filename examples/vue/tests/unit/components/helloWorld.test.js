// import { render } from '@testing-library/vue';
import { shallowMount as render } from '@vue/test-utils';

import HelloWorld from '@/components/HelloWorld.vue';

describe('HelloWorld.vue', () => {
  const setupWrapper = async function (props) {
    return await render(HelloWorld, { props });
  };

  test('Renders message', async () => {
    const props = {
      message: 'Hello, world'
    };
    const wrapper = await setupWrapper(props);

    expect(2)
      .toEqual(3);

    expect(wrapper.html())
      .toMatchSnapshot();
  });
});
