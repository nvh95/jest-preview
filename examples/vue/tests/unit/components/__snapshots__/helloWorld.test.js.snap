// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue Renders message 1`] = `
<div class="hello">
  <h1>Hello, world</h1>
</div>
`;
